_posts/2015-06-12-apac-day-of-coderetreat-on-jun-13.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: post
+author: JosephYao
+authorname: Joseph Yao
+title: APAC Day of Coderetreat on Jun 13
+preview: 'As you may know, software craftsman and coderetreat culture is much weaker in APAC (Especially in China) than US and Europe. So, I got an idea to organize a day event of coderetreat (like GDCR) to promote that culture.'
+---
+
+
+As you may know, software craftsman and coderetreat culture is much weaker in APAC (Especially in China) than US and Europe. So, I got an idea to organize a day event of coderetreat (like GDCR) to promote that culture. After I talked to other GDCR organisers in China and Asia, the idea was accepted quite well. Then, the date was set at Jun 13 (Saturday).
+
+
+We now have 14 cities registered for this event and I created a map  [here](https://www.google.com/maps/d/viewer?mid=zFRRheM6-9AQ.k8wlhsC1wCNY&amp;usp=sharing).  
+
+
+![embedded map](https://www.google.com/maps/d/u/0/embed?mid=1lsAFrsPkiHnp0vUpqW_50NJQiuI)
```

Comment:
We can keep the `iframe` here so the map appears in the article, right now (see [here](https://deploy-preview-59--hairdresser-umbrellas-72171.netlify.com/blog/2015/06/12/apac-day-of-coderetreat-on-jun-13/)) the article doesn't really look too nice